DariaCagle3(obligatory).cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+
+enum TASKS { REVERSESTR = 1, ARRINSERT };
+
+void reverseString() {
+  std::cout << "Input a string:\n";
+  char arr[255]{};
+  std::cin >> arr;
+  int length = strlen(arr);
+  char revArr[255]{};
+  for (int ch = 0; ch < length; ch++) {
+    revArr[ch] = arr[length - ch - 1];
+  }
+  std::cout << revArr << "\n";
+}
+
+void insertInArray() {
+  constexpr int length = 5;
+  int arr[length]{};
+  int countZero = 0;
+  while (countZero < length) {
+    std::cout << "Input a number:\n";
+    int number;
+    std::cin >> number;
+    int i = countZero;
+    arr[i] = number;
+    while (i > 0) {
+      if (arr[i] < arr[i - 1]) {
+        int temp = arr[i - 1];
+        arr[i - 1] = arr[i];
+        arr[i] = temp;
+      }
```

Comment:
мне кажеться что такой цикл немного выглядит сумбурно
```suggestion
   std::cin >> arr[countZero];
   for (size_t i {countZero}; i > 0; --i) {
      if (arr[i] < arr[i - 1]) {
        int temp = arr[i - 1];
        arr[i - 1] = arr[i];
        arr[i] = temp;
      }
    }
```

Reply:
как я могу константную переменную вносить в фор, если она кое-что делает и раньше? там два цикла, и в обоих используется эта переменная.

Reply:
исправила